e2e/angular-core/src/angular-linting.test.ts

```diff
@@ -7,73 +7,63 @@ import {
 } from '@nrwl/e2e/utils';
 import * as path from 'path';
 
-describe('Angular Package', () => {
-  describe('linting', () => {
-    beforeAll(() => newProject());
-    afterAll(() => cleanupProject());
+describe('linting', () => {
+  beforeAll(() => newProject());
```

Comment:
Can we combine linting with some other suite so that we can reuse the same `newProject` and potentially also the same `app` and `lib`?